src/modules/pos/validator/components/EditStake/index.js

```diff
@@ -201,6 +239,25 @@ const EditStake = ({ history, stakeEdited, network, staking }) => {
                   <span className={styles.space} />
                 </>
               )}
+              <div className={styles.durationSelect}>
+                <div>
+                  <span>{t('You will get')}</span>
+                  <RewardDurationSwitch
+                    value={selectedDuration}
+                    changeCategory={handleSelectDuration}
+                    categories={[
+                      { value: REWARD_DURATIONS.monthly, label: t('Monthly') },
+                      { value: REWARD_DURATIONS.yearly, label: t('Yearly') },
+                    ]}
+                  />
+                  <span>:</span>
+                </div>
+                {!isFetchingRewards ? (
+                  <TokenAmount val={expectedReward?.data.reward || 0} token={token} />
```

Comment:
Shall we not default to `-` here instead of `0`

Reply:
This will be a major change since this component is reused everywhere? maybe it should be handled separately after discussing the approach?